docs/admin/troubleshooting.md

```diff
@@ -9,3 +9,15 @@ The marketplace rest server and webportal url is aligned with OpenPAI cluster. I
 In Marketplace v1.6.0, when the admin set `cluster.common.marketplace = "true"`, the OpenPAI deploy script will auto add a marketplace item into the webportal's plugin.
 Upgrade marketplace from previous version without changing the `services-configuration.yaml` file, the auto added marketplace item may cause duplicate marketplace tags in webportal [#230](https://github.com/microsoft/openpaimarketplace/issues/230).
 Admin can either set `webportal.marketplace: false` to avoid the auto added item, or remove the old marketplace item from `webportal.plugins` to delete the old one.
+
+## How to migrate marketplace item schema from v1.5.0 to v1.6.0
```

Comment:
should we add this in trouble shooting also

Reply:
Is there a troubleshooting document other than docs/admin/troubleshooting.md ?